.taskcluster.yml

```diff
@@ -149,13 +149,21 @@ tasks:
             git checkout {{event.head.sha}}
             source $HOME/servotidy-venv/bin/activate
             servo-tidy
+            sccache --stop-server || true
```

Comment:
You might want to stick a `--stop-server` invocation after your test steps as well, since sccache outputs stats after shutting down and those are useful to have in build logs.